salt/states/file.py

```diff
@@ -1062,7 +1065,7 @@ def _get_template_texts(source_list=None,
         log.debug(msg.format(rndrd_templ_fn, source))
         if rndrd_templ_fn:
             tmplines = None
-            with salt.utils.fopen(rndrd_templ_fn, 'rb') as fp_:
+            with salt.utils.files.fopen(rndrd_templ_fn, 'rb') as fp_:
```

Comment:
i think you may have unintentionally leaked some changes from develop in here. You might want to review again.